src/main/sensors/compass.h

```diff
@@ -48,15 +49,16 @@ extern mag_t mag;
 typedef struct compassConfig_s {
     int16_t mag_declination;                // Get your magnetic decliniation from here : http://magnetic-declination.com/
                                             // For example, -6deg 37min, = -637 Japan, format is [sign]dddmm (degreesminutes) default is zero.
-    sensor_align_e mag_align;               // mag alignment
+    flightDynamicsTrims_t magZero;
+    sensorAlignment_t mag_customAlignment;
```

Comment:
This has to go at the end, or the config will be corrupted.

See https://github.com/betaflight/betaflight/blob/master/docs/development/ParameterGroups.md

Reply:
yes if the version isn't changed and other cleanups reverted.  Let me know if you want the version change and other cleanups done or just to place the new setting at the end.